docs/src/user-guide/guides-using-presto.md

```
@@ -31,8 +31,31 @@ Using Presto with CLP requires:
 
-Follow the [quick-start](./quick-start/index.md) guide to set up CLP and compress your logs. A
-sample dataset that works well with Presto is [postgresql].
+1. Follow the [quick-start](./quick-start/index.md) guide to download and extract the CLP package,
+   but don't start the package just yet.
+2. Before starting the package, update the package's config as follows:
+
+    * Open `etc/clp-config.yml` located within the package.
+    * Uncomment the `database` section.
+    * Change `database.host` value to a non-localhost hostname/IP.
+    * After the change, the `database` section should look something like this:
+      ```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)